src/index.js

```diff
@@ -45,6 +43,27 @@ const createList = (todoItem) => {
   todoStatusUpdate(tasks);
 };
 
+addTodo(tasks);
+deleteCompletedTodoItem(tasks);
```

Comment:
- [ ] Please add functionality to delete a single todo item. See [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Implement%20a%20function%20for%20deleting%20a%20task%20(remove%20an%20element%20from%20the%20array).).
- [ ] Also ensure that deleting a task should update all remaining items' indexes, so they represent the current list order and are unique. See [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=Deleting%20a%20task%20should%20update%20all%20remaining%20items%27%20indexes%2C%20so%20they%20represent%20the%20current%20list%20order%20and%20are%20unique.).
- [ ] When an item gets deleted also ensure that the change is updated in localStorage. See [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=All%20changes%20to%20the%20To%20Do%20List%20should%20be%20saved%20in%20local%20storage.).